slave/buildslave/monkeypatches/bug4881.py

```diff
@@ -14,6 +14,8 @@
 #
 # Copyright Buildbot Team Members
 
+from builtins import range
```

Comment:
It allows us to use xrange in py2, and range in py3. It "backports" py3 range to py2. You can read more about it in the [python-future documentation](http://python-future.org/imports.html?highlight=builtins)